test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -550,12 +547,19 @@ Address RandomCodeOptions::getRandomAddress(AddressType _type) const
 	}
 }
 
-void RandomCodeOptions::setWeights()
+int RandomCode::weightedOpcode(std::vector<int>& _weights)
```

Comment:
Can this function receive `opCodeProbability` instead of `_weights`?  Then, you don't need the static variable.  Static variables are like global variables, so let's try not to introduce them.

Reply:
Alternatively, maybe you can make `opCodeProbability` a local variable in this function.  That seems cleaner.

Reply:
from where should it receve opCodeProbability ?

Reply:
the idea is that random generators are managed by random code class.

Reply:
The caller.  Read my second comment as well, and choose one of my two suggestions (or something else of course).